docs/src/components/landing-page/Banner.js

```diff
@@ -0,0 +1,116 @@
+ * The above copyright notice and this permission notice shall be included in all *
+ * copies or substantial portions of the Software.                                *
+ *                                                                                *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR     *
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,       *
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE    *
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER         *
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,  *
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE  *
+ * SOFTWARE.                                                                      *
+ **********************************************************************************/
+
+import ButtonLink from 'components/ButtonLink'
+import { useEffect, useState } from 'react'
+import StarBlurSvg from '../../../public/star-blur.svg'
+import WorldDotSvg from '../../../public/world-dot.svg'
+import { siteConfig } from 'siteConfig'
+
+export default function Banner(props) {
+  return (
+    <div
+      className={`relative flex flex-col py-32 ${
+        props.className ? props.className : ''
+      }`}
+    >
+      <div className="z-10 m-auto mb-0 px-4 max-w-4xl text-5xl font-bold text-center text-white">
+        <h1>
+          Know when your{' '}
+          <span className="bg-gradient-to-r from-purple-monika to-aqua-monika bg-clip-text text-transparent">
+            web app is down
+          </span>{' '}
+          before your users do.
+        </h1>
+      </div>
+      <div className="z-10 mt-4 px-4 font-normal text-center text-white">
+        <p>
+          React faster when your app is having problem before your users notice!
+        </p>
+        <p>Let's get started in seconds.</p>
+      </div>
+      <div className="z-10 m-auto mt-4">
+        <p className="mt-4 px-16 py-4 bg-gray-monika bg-opacity-10 rounded-md">
+          <span className="bg-gradient-to-r from-purple-monika to-aqua-monika bg-clip-text font-normal text-center text-transparent">
+            npm i -g @hyperjumptech/monika
+          </span>
+        </p>
+        <div className="flex flex-row justify-between mt-4">
+          <div className="flex">
+            <ButtonLink
+              className="w-40 leading-snug font-semibold text-center"
+              href="/quick-start"
+              rel="noopener noreferrer"
+            >
+              Get Started!
+            </ButtonLink>
+          </div>
+          <div className="flex">
+            <StarButton />
+          </div>
+        </div>
+      </div>
+      <img src={StarBlurSvg} className="absolute -top-16 right-0" />
+      <img
+        src={WorldDotSvg}
+        className="sm:invisible xl:visible absolute -bottom-128 -left-16"
```

Comment:
Please add `max-w-2xl` so that the image will not cover the npm install text.

Before:
<img width="1398" alt="Screenshot 2021-06-15 at 20 12 53" src="https://user-images.githubusercontent.com/311343/122102794-1c0abc00-ce16-11eb-83fe-d2e1c76d4bd2.png">


After:
<img width="1385" alt="Screenshot 2021-06-15 at 20 12 01" src="https://user-images.githubusercontent.com/311343/122102690-fc739380-ce15-11eb-810c-5305b159b0f3.png">